### ISSUE TITLE
SC3014: Offer a quickfix, as it is trivial

### ISSUE THREAD

**Issue by stdedos** [koalaman/shellcheck#2661]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC3014
- My shellcheck version (`shellcheck --version` or "online"): 0.9.0
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related https://github.com/koalaman/shellcheck/issues?q=is%3Aissue+SC3014+offer+fix

#### Here's a snippet or screenshot that shows the problem:

https://github.com/koalaman/shellcheck/blob/d1d574c0911bb9405ffb79926e6d4dd4c0a16ca5/src/ShellCheck/Checks/ShellSupport.hs#L217-L218

#### Here's what shellcheck currently says:

```
In input.sh line 10:
    if [ "$#" == 0 ] ; then
              ^-- SC3014 (warning): In POSIX sh, == in place of = is undefined.

For more information:
  https://www.shellcheck.net/wiki/SC3014 -- In POSIX sh, == in place of = is ...
```

#### Here's what I wanted or expected to see:

... "fix" also

https://github.com/koalaman/shellcheck/blob/d1d574c0911bb9405ffb79926e6d4dd4c0a16ca5/src/ShellCheck/Checks/ShellSupport.hs#L217-L218

```hs
 bashism (TC_Binary id SingleBracket "==" _ _) = 
         warnWithFix id 3014 "== in place of = is" (replaceToken id params "==" "=")
```

... or something equivalent/correct 😅 🥵
